pkg/cloud/cloud.go

```diff
@@ -129,16 +126,16 @@ type cloud struct {
 // NewCloud returns a new instance of AWS cloud
 // It panics if session is invalid
 func NewCloud(region string) (Cloud, error) {
-	awsConfig := &aws.Config{
-		Region:                        aws.String(region),
-		CredentialsChainVerboseErrors: aws.Bool(true),
```

Comment:
Is there a `CredentialsChainVerboseErrors` equivalent in v2?

Reply:
I couldn't find one, but could find https://github.com/aws/aws-sdk-go-v2/issues/98 so I don't think there is an alternative at this stage.

Reply:
hm, yeah it doesn't seem like there's one. Thanks for looking into this